css-view-transitions-2/Overview.bs

```diff
@@ -205,6 +206,23 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		```
 	</div>
 
+# Pseudo-classes # {#pseudo-classes}
+
+## Active View Transition Pseudo-class '':active-view-transition()''' ## {#the-active-view-transition-pseudo}
+
+The <dfn id='active-view-transition-pseudo'>:active-view-transition(<<vt-type-selector>>)</dfn> pseudo-class applies to the root element of the document, if it has a matching [=active view transition=].
+It has the following syntax definition:
+
+<pre class=prod>
+	:active-view-transition(<<vt-type-selector>>)
+	<dfn>&lt;vt-type-selector></dfn> = '*' | <<custom-ident>>#
+</pre>
+
+An ''::active-view-transition()'' pseudo-class matches the [=document element=] when it has an non-null [=active view transition=] |viewTransition|, for which any of the following are true:
+
+* The <<vt-type-selector>> is ''*''
+* |viewTransition|'s [=ViewTransition/active types=] is null
```

Comment:
Can you file a separate issue for this? We haven't explicitly discussed this, and my intuition is that it shouldn't match if type-selector is non-`*` and `active types` is null

Reply:
You're right. I think this line should go, will open an issue.